cmd/bootkube/render.go

```diff
@@ -21,6 +21,7 @@ const (
 	apiOffset                    = 1
 	dnsOffset                    = 10
 	etcdOffset                   = 15
+	bootEtcdOffset               = 200
```

Comment:
> we don't know what the pod IP will be

That's why we should use downward API. First of all, it doesn't need to reserve a service IP. Second, it's more generic on k8s platform than this method.